app/src/features/rules/components/RulesList/components/RulesListIndex/RulesListIndex.tsx

```diff
@@ -176,8 +176,8 @@ const RulesList: React.FC<Props> = () => {
   );
 
   const handleNewRuleOnClick = useCallback(
-    async (ruleType?: RuleType) => {
-      if (ruleType) trackRuleCreationWorkflowStartedEvent(ruleType, "screen");
+    async (ruleType?: RuleType, source = "rule_table_create_new_rule_button") => {
```

Comment:
would be better to make constants for these sources

Reply:
++